transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -319,4 +379,47 @@ public IovArray iovArray() {
         }
         return iovArray;
     }
+
+    private void registerFixedBuffers(int ringFd) {
+        if (buffersRegistered) {
+            // Need to unregister first
+            int ret = Native.ioUringRegister(ringFd, Native.IORING_UNREGISTER_BUFFERS,
+                    0L /*NULL*/, 0);
+            if (ret < 0 && ret != Native.ERRNO_ENXIO_NEGATIVE) {
+                logger.warn("UNREGISTER_BUFFERS returned " + ret); //TODO
+                return;
+            }
+            buffersRegistered = false;
+        }
+        int bufCount = tracker.getCount();
+        if (bufCount > 0) {
+            final ByteBuf ioVecs = tracker.getIoVecs();
+            try {
+                bufCount = tracker.getCount(); // note this may have changed due to the iovec alloc
+                int ret = Native.ioUringRegister(ringFd, Native.IORING_REGISTER_BUFFERS,
+                        ioVecs.memoryAddress(), bufCount);
+                if (ret < 0) {
+                    //TODO reset tracker indices here
+                    throw new RuntimeException("REGISTER_BUFFERS returned " + ret);  //TODO
+                }
+                buffersRegistered = true;
+            } finally {
+                ioVecs.release();
+            }
+        }
+    }
+
+    private void pauseLongIo(IOUringSubmissionQueue submissionQueue) {
+        for (IntObjectMap.PrimitiveEntry<AbstractIOUringChannel> ent : channels.entries()) {
+            ent.value().removePolls();
+        }
+        if (!pendingWakeup) {
+            submissionQueue.addReadCancel(eventfd.intValue());
```

Comment:
I dont think that we need that, as the socket are non blocking

Reply:
eventfd was changed to be blocking, so it's just needed for that